test/src/main/java/com/redhat/lightblue/test/AbstractCRUDTestController.java

```diff
@@ -150,6 +196,23 @@ protected JsonNode getLightblueMetadataJson() throws Exception {
     protected abstract JsonNode[] getMetadataJsonNodes() throws Exception;
 
     /**
+     * Sets if any access settings in metadata should be altered to 'anyone' for
+     * testing purposes. Defaults to <code>true</code>.
+     * @param grantAnyoneAccess
+     */
+    protected void setGrantAnyoneAccess(boolean grantAnyoneAccess) {
+        this.grantAnyoneAccess = grantAnyoneAccess;
+    }
+
+    /**
+     * @return <code>true</code> if access settings on metadata should be altered to
+     * 'anyone', otherwise <code>false</code>. Defaults to <code>true</code>.
+     */
+    public boolean IsGrandAnyoneAccess() {
```

Comment:
I suspect this should have been `IsGrantAnyoneAccess` (not "grand")..
Also, why capital "I" in "Is"?